src/main/java/codechicken/nei/BookmarkPanel.java

```diff
@@ -1245,64 +1275,124 @@ public void addOrRemoveItem(ItemStack stackover, final String handlerName, final
         if (slot != null && StackInfo.equalItemAndNBT(slot.item, stackover, true)) {
             BGrid.removeRecipe(slot.slotIndex, saveIngredients);
         } else {
-            final NBTTagCompound nbTagA = StackInfo.itemStackToNBT(stackover, saveStackSize);
-            final ItemStack normalizedA = StackInfo.loadFromNBT(nbTagA, saveStackSize ? nbTagA.getInteger("Count") : 0);
             BookmarkRecipeId recipeId = null;
 
-            if (handlerName != "" && ingredients != null) {
+            if (!handlerName.isEmpty() && ingredients != null) {
```

Comment:
results in https://github.com/GTNewHorizons/GT-New-Horizons-Modpack/issues/16763